sklearn/pipeline.py

```diff
@@ -253,6 +253,12 @@ def _fit(self, X, y=None, **fit_params):
         fit_params_steps = {name: {} for name, step in self.steps
                             if step is not None}
         for pname, pval in fit_params.items():
+            if '__' not in pname:
+                raise TypeError(
+                    "Passing a parameter directly to a pipeline isn't "
+                    "supported. You can pass parameters to specific "
+                    "steps of your pipeline, e.g. `pipe.fit (X, y, "
```

Comment:
Please remove blank space in call to `fit (`, also remove comma at the end.

Maybe add 

You can pass parameters to specific steps of your pipeline using the stepname__parameter format, e.g. ...